protos/backend_service.proto

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+syntax = "proto3";
+
+package durabletask.protos.backend.v1;
+
+option csharp_namespace = "Microsoft.DurableTask.Protobuf";
+option java_package = "com.microsoft.durabletask.implementation.protobuf";
+option go_package = "/internal/protos";
+
+import "orchestrator_service.proto";
+
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/wrappers.proto";
+import "google/protobuf/empty.proto";
+
+// gRPC service used by Durable Task Framework (DTFx) backend implementations.
+// The RPCs in this service are used by DTFx backends to manage orchestration state.
+service BackendService {
```

Comment:
Decided to revert to a single service definition instead of multiple. The main reason was the unnecessary code complexity across both SDK and service codebases. The secondary reason is that there's no plan to have separate client/worker SDKs that communicate with the backend service (separate client/worker services make sense for the engine API, but not the backend API).

Reply:
I wish this were discussed a bit more as it does make some scenarios more complicated...

Reply:
We should probably discuss this more then. You and I might have different ideas of what the scenarios are.